docs/admin.rst

```diff
@@ -62,6 +68,7 @@ admin class
         list_display = ["id", "name", "status"]
         history_list_display = ["status"]
         search_fields = ['name', 'user__username']
+        history__list_per_page = 100
```

Comment:
Im not exactly sure why you're not seeing the pagination because I just copied in what SimpleHistory as it is in my commits here into our own code and it seems to be working just fine with the pagination. I copied in both the object_history.html and SimpleHistoryAdmin class

Reply:
So if I look at the files changed in this PR, I see no html file added or changed. And I guess the change as in the link I showed should be in `_object_history_list.html` for it to work.